provider/resource.go

```diff
@@ -43,88 +43,89 @@ const (
-	azureRecoveryServices    = "RecoveryServices"    // Recovery Services
-	azureRedis               = "Redis"               // RedisCache
-	azureRelay               = "Relay"               // Relay
-	azureSecurityCenter      = "SecurityCenter"      // Security Center
-	azureSentinel            = "Sentinel"            // Sentinel
-	azureServiceBus          = "ServiceBus"          // ServiceBus
-	azureServiceFabric       = "ServiceFabric"       // Service Fabric
-	azureSearch              = "Search"              // Search
-	azureSignalr             = "SignalR"             // SignalR
-	azureSiteRecovery        = "SiteRecovery"        // SiteRecovery
-	azureSQL                 = "Sql"                 // SQL
-	azureStorage             = "Storage"             // Storage
-	azureStreamAnalytics     = "StreamAnalytics"     // StreamAnalytics
-	azureSynapse             = "Synapse"             // Synapse
-	azureWaf                 = "Waf"                 // WAF
+	advisor                    = "Advisor"               // Advisor
+	azureAnalysisServices      = "AnalysisServices"      // Analysis Services
+	azureAPIManagement         = "ApiManagement"         // API Management
+	azureAppConfiguration      = "AppConfiguration"      // App Configuration
+	azureAppInsights           = "AppInsights"           // AppInsights
+	azureAppPlatform           = "AppPlatform"           // AppPlatform
+	azureAppService            = "AppService"            // App Service
+	azureAttestation           = "Attestation"           // Attestation
+	azureAutomation            = "Automation"            // Automation
+	azureAuthorization         = "Authorization"         // Authorization
+	azureBackup                = "Backup"                // Backup
+	azureBatch                 = "Batch"                 // Batch
+	azureBlueprint             = "Blueprint"             // Blueprint
+	azureBot                   = "Bot"                   // Bot
+	azureCDN                   = "Cdn"                   // CDN
+	azureCognitive             = "Cognitive"             // Cognitive
+	azureCompute               = "Compute"               // Virtual Machines
+	azureContainerService      = "ContainerService"      // Azure Container Service
+	azureCore                  = "Core"                  // Base Resources
+	azureCosmosDB              = "CosmosDB"              // Cosmos DB
+	azureCostManagement        = "CostManagement"        // CostManagement
+	azureDashboard             = "Dashboard"             // Dashboard
+	azureDatabaseMigration     = "DatabaseMigration"     // Database Migration
+	azureDataFactory           = "DataFactory"           // Data Factory
+	azureDatalake              = "DataLake"              // Data Lake
+	azureDataShare             = "DataShare"             // DataShare
+	azureDataBricks            = "DataBricks"            // DataBricks
+	azureDesktopVirtualization = "DesktopVirtualization" // DesktopVirtualization
```

Comment:
I believe the service is "Virtual Desktop" - the same as the TF name. Why did you pick "Desktop Virtualization"?

Reply:
I picked it as that was in the TF module - https://github.com/terraform-providers/terraform-provider-azurerm/pull/8605/files#diff-d626fff06c84ff73c96b49db331f82ceR33

Will change it now though :)

Reply:
I think they are right, the same name is used in Open API specs: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/desktopvirtualization/resource-manager

Let's go ahead with the current version, sorry for the confusion!

Reply:
Ah I forgot to change this back - I will do this now